### PR TITLE
BACK-16

### PR DESCRIPTION
**Pull request by larasodjati**
Added .env.example file as a reference

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -0,0 +1,10 @@
+MONGO_URI=mongodb+srv://LVien:<password>@practicum.twrhx3o.mongodb.net/databasename?retryWrites=true&w=majority
+("url from mongodb connection")
+
+JWT_SECRET=NcRfUjXn2r5u8x/A%D*G-KaPdSgVkYp3
```
Comment (jpremo) [inline .env.example:4]: Can you not include any secret value here? We want everyone to have to generate their own. Having this here might encourage someone to use this value which would not be preferable. You can just substitute it with "SECRET_VALUE" or something along those lines.

File: .env.example [hunk 2]
```diff
@@ -0,0 +1,10 @@
+MONGO_URI=mongodb+srv://LVien:<password>@practicum.twrhx3o.mongodb.net/databasename?retryWrites=true&w=majority
```
Comment (jpremo) [inline .env.example:1]: LVien is the username here right? Can you write it similar to how you did for password (<username>). Similarly I would recommend hiding practicum.twrhx3o.mongodb.net as well.

**Review by jpremo — CHANGES_REQUESTED** [review]

File: .env.example (hunk 1 above)
Comment (larasodjati) [reply .env.example:4]: Noted. I have made some changes to that file so the secret value will be stated as anonymous.

Comment (larasodjati) [reply .env.example:4]: `JWT_SECRET=SECRET_VALUE
("signature or a code that is used to identify the authenticity of the token,if you want to generate a secure key you can go to 
https://allkeysgenerator.com/, choose encryption key then 256bit as security level, the copy and paste the key here")`

File: .env.example (hunk 2 above)
Comment (larasodjati) [reply .env.example:1]: Noted. I fixed the code as below : 
`MONGO_URI=mongodb+srv://<username>:<password>@xxxxxx.xxxxx.mongodb.net/database name?retryWrites=true&w=majority
("url from mongodb connection")`

**Review by jpremo — APPROVED** [review]

**Pull request merged**